CHANGELOG.md

```diff
@@ -7,16 +7,14 @@
 ### Bug Fixes:
 
 1. Fixed issue in **resource_aviatrix_firewall_instance_association** and **resource_aviatrix_gateway** for Azure where we no longer require special handling of ``fqdn_lan_interface``  and ``lan_interface``.
-4. Fixed issue in **aviatrix_edge_platform_device_onboarding** where importing was failing.
-
+2. Fixed issue in **aviatrix_edge_platform_device_onboarding** where importing was failing.
 ### Features:
 #### Provider:
 1. Added support for the Terraform provider to properly set the user-agent when making requests.
 
 ### Deprecations:
 
 1. Deprecated ``http_access`` in **aviatrix_controller_config**. This configuration value no longer has any effect. It will be removed from the Aviatrix provider in the 3.2.0 release.
-2. Deprecated ``bandwidth`` in WAN/LAN/MGMT interfaces in the following resources. This configuration value no longer has any effect. It will be removed from the Aviatrix provider in the 3.2.0 release.
   - **aviatrix_edge_csp**
   - **aviatrix_edge_csp_ha**
```

Comment:
need to delete the rest of this list :) the bullet point includes the list of resources